src/charts/bar.js

```diff
@@ -164,7 +166,16 @@ define(function(require) {
          * @param {BarChartData} _data The data to attach and generate the chart
          */
         function exports(_selection) {
-            _selection.each(function(_data) {
+            if (locale && enableLabels) {
+                setDefaultLocale(locale)
+                    .then(() => initChart(_selection))
```

Comment:
Can initialization of the locale be done somewhere else?

Reply:
Sure, i first had this in the drawLabels call. What place suits you better ?